packages/simcore-sdk/src/simcore_sdk/nodeports/filemanager.py

```diff
@@ -0,0 +1,65 @@
+import os
+from pathlib import Path
+import shutil
+import logging
+import tempfile
+import tenacity
+from simcore_sdk.config.s3 import Config as s3_config
+from s3wrapper.s3_client import S3Client
+
+_LOGGER = logging.getLogger(__name__)
+_INTERNAL_DIR = os.path.join(tempfile.gettempdir(), "simcorefiles")
+
+class S3Settings(object):
+    def __init__(self):
+        self._config = s3_config()
+        self.client = S3Client(endpoint=self._config.endpoint,
+            access_key=self._config.access_key, secret_key=self._config.secret_key)
+        self.bucket = self._config.bucket_name
+        self.client.create_bucket(self.bucket)
+
+@tenacity.retry(stop=tenacity.stop_after_attempt(3) | tenacity.stop_after_delay(10))

```

Comment:
Really cool! tenaticy! I guess you can use ``or`` instead of ``|`` operator ??

Reply:
probably